checkout-ui-custom/src/controller/CollectController.js

```diff
@@ -160,6 +115,28 @@ const CollectController = (() => {
       .click(() => iframeFunctions(MANUAL));
   };
 
+  const resetPickup = () => {
+    $('.delivery-group-content').empty();
+    $('.btn-go-to-payment-wrapper').empty();

```

Comment:
I reviewed the workspace and it does seem to work, but I worry about these two lines as they still remove nodes from the dom and could cause issues

Reply:
Agreed, we have to review use of .remove(), .empty() and .html() which removes elements. I think we should take some time on that in a separate release, because `display:none` might not work for all scenarios.